trending_data/src/main/webapp/script.js

```diff
@@ -12,7 +13,7 @@ function initMap() {
     zoom: 3,
     minZoom: 2,
   });
-
+  infoWindws = new UniqueWindowHandler(map);
```

Comment:
I was referring to the variable name not the class name in the previous PR review comment